.github/CODEOWNERS

```diff
@@ -0,0 +1,4 @@
+# DOCUMENTATION -------------------------------------------------------
+# Docs folder ownership
+/docs/ @nextflow-io/docs
+/.github/workflows/ @nextflow-io/docs
```

Comment:
is `github/workflows` really needed?

Reply:
Not really - I was using the Platform repo as reference and they had it there. Both workflows in this repo are to do with publishing the docs (`Test docs build` and `Check for spelling errors`) so I figured why not. But happy to remove if you'd prefer.

Reply:
I'd not expect docs team to work directly on GH actions. I'd keep just `/docs/`

Reply:
```suggestion
```

Reply:
Done ✅